docs/troubleshooting/remote-troubleshooting.md

```diff
@@ -0,0 +1,282 @@
+```
+
+You can inspect the details of a specific resource request with the `-long` option:
+
+```console
+$ condor_q -all -name htcondor-ce.chtc.wisc.edu -pool htcondor-ce.chtc.wisc.edu:9619 -l 24631.5
+Arguments = ""
+BufferBlockSize = 32768
+BufferSize = 524288
+BytesRecvd = 58669.0
+BytesSent = 184201.0
+ClusterId = 24631
+Cmd = "DIRAC_Ullq7V_pilotwrapper.py"
+CommittedSlotTime = 0
+CommittedSuspensionTime = 0
+CommittedTime = 0
+```
+
+### Retrieving HTCondor-CE configuration ###
+
+To verify a remote HTCondor-CE configuration,
+use [condor_config_val](https://htcondor.readthedocs.io/en/latest/man-pages/condor_config_val.html):
+
+```console
+$ condor_config_val -name htcondor-ce.chtc.wisc.edu -pool htcondor-ce.chtc.wisc.edu:9619 -dump
+# Configuration from master on htcondor-ce.chtc.wisc.edu <128.104.100.65:9619?addrs=128.104.100.65-9619+[2607-f388-107c-501-216-3eff-fe89-aa3]-9619&alias=htcondor-ce.chtc.wisc.edu&noUDP&sock=master_1744571_b0a0>
+ABORT_ON_EXCEPTION = false
+ACCOUNTANT_HOST = 
+ACCOUNTANT_LOCAL_DOMAIN = 
+ActivationTimer = ifThenElse(JobStart =!= UNDEFINED, (time() - JobStart), 0)
+ActivityTimer = (time() - EnteredCurrentActivity)
+ADD_WINDOWS_FIREWALL_EXCEPTION = $(CondorIsAdmin)
+ADVERTISE_IPV4_FIRST = $(PREFER_IPV4)
+ALL_DEBUG = D:CAT D_ALWAYS:2
+ALLOW_ADMIN_COMMANDS = true
+```
+
+If you know the name of the configuration variable, you can query for it directly:
+
+```console
+$ condor_config_val -name htcondor-ce.chtc.wisc.edu -pool htcondor-ce.chtc.wisc.edu:9619 -verbose JOB_ROUTER_SCHEDD2_NAME
+JOB_ROUTER_SCHEDD2_NAME = htcondor-ce.chtc.wisc.edu
+ # at: /etc/condor-ce/config.d/02-ce-condor.conf, line 20
+ # raw: JOB_ROUTER_SCHEDD2_NAME = $(FULL_HOSTNAME)
+```
+
+Verifying Resource Request Submission
+-------------------------------------
+
+After verifying that all the remote HTCondor-CE [daemons are up](#inspecting-daemons),
+you can start submitting resource requests!
+
+### Verifying authentication ###
+
+Before submitting a successful resource request, you will want to verify that you have submit privileges.
+For this, you will need a credential such as a grid proxy:
+
+```console
+$ voms-proxy-info
+subject   : /DC=org/DC=cilogon/C=US/O=University of Wisconsin-Madison/CN=Brian Lin A2266246/CN=41319870
+issuer    : /DC=org/DC=cilogon/C=US/O=University of Wisconsin-Madison/CN=Brian Lin A2266246
+identity  : /DC=org/DC=cilogon/C=US/O=University of Wisconsin-Madison/CN=Brian Lin A2266246
+type      : RFC compliant proxy
+strength  : 1024 bits
+path      : /tmp/x509up_u1000
+timeleft  : 3:55:22
+```
+
+After you have retrieved your credential, verify that you have the ability to submit requests to the remote HTCondor-CE
+(i.e., `WRITE` access) with [condor_ping](https://htcondor.readthedocs.io/en/latest/man-pages/condor_ping.html):
+
+```console
+$ _condor_SEC_CLIENT_AUTHENTICATION_METHODS=SCITOKENS,GSI
+$ _condor_SEC_TOOL_DEBUG=D_SECURITY:2  # Extremely verbose debugging for troubleshooting authentication issues
```

Comment:
```suggestion
$ export _condor_SEC_CLIENT_AUTHENTICATION_METHODS=SCITOKENS,GSI
$ export _condor_SEC_TOOL_DEBUG=D_SECURITY:2  # Extremely verbose debugging for troubleshooting authentication issues
```
These will have no effect otherwise.
```
$ _condor_SEC_TOOL_DEBUG=D_SECURITY condor_ping
```
_only_ works because they're on the same line.